docs/source/conf.py

```diff
@@ -320,6 +320,7 @@
     "Quantize",
     # torch.utils.backcompat
     "Warning",
+    "SymbolicIntNode"
```

Comment:
We should NOT add anything to this list.
If this is public, then it should have a proper entry on the website.
If it is not public, it should be marked as such and have a leading `_`.

Reply:
cc @Krovatkin  - should we change this? 

This was just a reland PR, the original PR was already stamped.

Reply:
@albanD it's not public. I'll add an underscore.

Reply:
Thanks!